tasks/src/main/scala/dagr/tasks/fgbio/CallDuplexConsensusReads.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package dagr.tasks.fgbio
+
+import dagr.core.tasksystem.Pipe
+import dagr.tasks.DagrDef.PathToBam
+import dagr.tasks.DataTypes.SamOrBam
+
+import scala.collection.mutable.ListBuffer
+
+class CallDuplexConsensusReads(val in: PathToBam,
+                               val out: PathToBam,
+                               val readNamePrefix: Option[String] = None,
+                               val readGroupId: Option[String] = None,
+                               val errorRatePreUmi: Option[Int] = None,
+                               val errorRatePostUmi: Option[Int] = None,
+                               val minInputBaseQuality: Option[Int] = None
```

Comment:
Lol.  I'll align these.  I don't think we have a _rule_ per se.  I just tend to align things when it bothers me 😛